code/typescript/pipelines-workshop/package.json

```diff
@@ -1,33 +1,33 @@
 {
-    "name": "cdk-workshop",
-    "version": "0.1.0",
-    "main": "bin/index.js",
-    "types": "bin/index.d.ts",
-    "bin": {
-        "cdk-workshop": "bin/cdk-workshop.js"
-    },
-    "scripts": {
-        "build": "tsc",
-        "watch": "tsc -w",
-        "cdk": "cdk"
-    },
-    "devDependencies": {
-        "@types/jest": "^26.0.10",
-        "@types/node": "^10.17.28",
-        "typescript": "^3.9.7",
-        "@aws-cdk/assert": "1.60.0",
```

Comment:
We don't need the assertion library or jest types because pipelines-workshop does not cover testing. These are only present in the tests-workshop now.